### PR TITLE
google-apps-script: add missing getForegroundColorObject() to TextStyle class in the spreadsheet.d.ts

### PR DESCRIPTION
**Pull request by rcfaria01**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://developers.google.com/apps-script/reference/spreadsheet/text-style>

### CODE REVIEW

File: types/google-apps-script/google-apps-script.spreadsheet.d.ts [hunk 1]
```diff
@@ -2056,6 +2056,7 @@ declare namespace GoogleAppsScript {
       getFontFamily(): string | null;
       getFontSize(): Integer | null;
       getForegroundColor(): string | null;
+      getForegroundColorObject(): Color;
```
Comment (mtgto) [inline types/google-apps-script/google-apps-script.spreadsheet.d.ts:2059]: TextStyle#getForegroundColorObject returns `Color | null`.

```suggestion
      getForegroundColorObject(): Color | null;
```

Code example:

```js
function test() {
  const color = SpreadsheetApp.newTextStyle().build().getForegroundColorObject(); // null
  Logger.log(color);
}
```

Comment (rcfaria01) [reply types/google-apps-script/google-apps-script.spreadsheet.d.ts:2059]: I really like your suggestion. Thank you @mtgto. I was pondering how to get a test wrapper to add a test for the method. I will make the necessary changes today and add the test.

Comment (rcfaria01) [reply types/google-apps-script/google-apps-script.spreadsheet.d.ts]: All done. Also added the setForegroundColorObject method which was also missing (from TextStyleBuilder), and tests for both get and set methods.

Comment (mtgto) [reply types/google-apps-script/google-apps-script.spreadsheet.d.ts]: Thanks! 👍

**Review by mtgto — APPROVED** [review]
LGTM. Thank you for your contribution 👏

**Pull request merged**